src/routes/(inner)/elements/chat/+page.svelte

```diff
@@ -0,0 +1,563 @@
+					</p>
+					<p class="font-bold">Three Row Layout</p>
+					<CodeBlock
+						language="html"
+						code={`
+<div class="h-full grid grid-rows-[auto_1fr_auto] gap-1">
+	<div class="bg-surface-500/30 p-4">(search)</div>
+	<div class="bg-surface-500/30 p-4">(list)</div>
+	<div class="bg-surface-500/30 p-4">(footer)</div>
+</div>
+					`}
+					/>
+					<p class="font-bold">Two Row Layout</p>
+					<p>
+						Note that we've set <a href="https://tailwindcss.com/docs/overflow" target="_blank" rel="noreferrer">overflow-y-auto</a> on the chat
+						region to enable vertical scrolling.
+					</p>
+					<CodeBlock
+						language="html"
+						code={`
+<div class="h-full grid grid-rows-[1fr_auto] gap-1">
+	<div class="bg-surface-500/30 p-4 overflow-y-auto">(chat)</div>
+	<div class="bg-surface-500/30 p-4">(prompt)</div>
+</div>
+					`}
+					/>
+				</svelte:fragment>
+			</DocsPreview>
+		</section>
+		<section class="space-y-4">
+			<h2>Message Bubbles</h2>
+			<p>By mixing Skeleton features with Tailwind styling, we can iterate and create our message bubbles.</p>
+			<DocsPreview background="neutral">
+				<svelte:fragment slot="preview">
+					<!-- Conversation -->
+					<section bind:this={elemChat} class="max-h-[500px] p-4 overflow-y-auto space-y-4">
+						{#each messages.slice(0, 2) as bubble, i}
+							{#if bubble.host === true}
+								<div class="grid grid-cols-[auto_1fr] gap-2">
+									<Avatar src="https://i.pravatar.cc/?img={bubble.avatar}" width="w-12" />
+									<div class="card p-4 variant-soft rounded-tl-none space-y-2">
+										<header class="flex justify-between items-center">
+											<p class="font-bold">{bubble.name}</p>
+											<small class="opacity-50">{bubble.timestamp}</small>
+										</header>
+										<p>{bubble.message}</p>
+									</div>
+								</div>
+							{:else}
+								<div class="grid grid-cols-[1fr_auto] gap-2">
+									<div class="card p-4 rounded-tr-none space-y-2 {bubble.color}">
+										<header class="flex justify-between items-center">
+											<p class="font-bold">{bubble.name}</p>
+											<small class="opacity-50">{bubble.timestamp}</small>
+										</header>
+										<p>{bubble.message}</p>
+									</div>
+									<Avatar src="https://i.pravatar.cc/?img={bubble.avatar}" width="w-12" />
+								</div>
+							{/if}
+						{/each}
+					</section>
+				</svelte:fragment>
+				<svelte:fragment slot="source">
+					<p>First, we need source data to work with. You might fetch this from a remote server in a real world application.</p>
+					<CodeBlock
+						language="ts"
+						code={`
+let messages = [
+	{
+		id: 0,
+		host: true,
+		avatar: 48,
+		name: 'Jane',
+		timestamp: 'Yesterday @ 2:30pm',
+		message: lorem,
+		color: 'variant-soft-primary'
+	},
+	{
+		id: 1,
+		host: false,
+		avatar: 14,
+		name: 'Michael',
+		timestamp: 'Yesterday @ 2:45pm',
+		message: lorem,
```

Comment:
```suggestion
		message: "lorem",
```

Reply:
@ryceg `lorem` is a variable so I don't have to implement the lorem paragraph over and over. I think this would be a detrimental change....right? Am I missing something?